man/systemd.exec.xml

```diff
@@ -1234,13 +1234,15 @@
       <varlistentry>
         <term><varname>NoNewPrivileges=</varname></term>
 
-        <listitem><para>Takes a boolean argument. If true, ensures
-        that the service process and all its children can never gain
-        new privileges. This option is more powerful than the
-        respective secure bits flags (see above), as it also prohibits
-        UID changes of any kind. This is the simplest, most effective
-        way to ensure that a process and its children can never
-        elevate privileges again.</para></listitem>
+        <listitem><para>Takes a boolean argument. If true, ensures that the service
+        process and all its children can never gain new privileges. This option is more
+        powerful than the respective secure bits flags (see above), as it also prohibits
+        UID changes of any kind. This is the simplest most effective way to ensure that a
+        process and its children can never elevate privileges again. Defaults to true in
+        the user manager instance if <varname>SystemCallFilter=</varname>,
+        <varname>SystemCallArchitectures=</varname>, or
+        <varname>RestrictAddressFamilies=</varname> are specified, and false otherwise.
+        </para></listitem>
```

Comment:
You might be right. But I don't understand the relationship with selinux well enough. If you could prep a patch that would be great.